packages/vscode-extension/src/extension.ts

```diff
@@ -55,7 +55,7 @@ import { loadLocalizedStrings, localize } from "./utils/localizeUtils";
 export let VS_CODE_UI: VsCodeUI;
 
 export async function activate(context: vscode.ExtensionContext) {
-  VsCodeLogInstance.info(localize("teamstoolkit.common.activate"));
+  VsCodeLogInstance.info("Teams Toolkit extension is now active!");
```

Comment:
Can't get localized string since extension variables haven't been loaded. Also I think no need to localize here.

Reply:
If the string is not used any more, you might also remove it from `package.nls.json`.

Reply:
Updated